OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/remote/DbxUtil.java

```diff
@@ -36,23 +37,33 @@ public static DbxClient getDbxClient() {
             );
             return null;
         }
-        return new DbxClient(getDbxConfig(), accessToken); 
+        return new DbxClientV2(getDbxConfig(), accessToken); 
     }
     
     public static DbxRequestConfig getDbxConfig() {
-        return new DbxRequestConfig("Open LaTex Studio",
-            Locale.getDefault().toString());
+        return DbxRequestConfig
+                .newBuilder("Open LaTex Studio")
+                .withUserLocale(Locale.getDefault().toString())
+                .build();
     }
     
     public static File downloadRemoteFile(DbxEntryDto remoteEntry, String localPath) {
-        DbxClient client = getDbxClient();
+        return downloadRemoteFile(remoteEntry.getRevision(), localPath);
+    }
+    
+    public static File downloadRemoteFile(FileMetadata metadata, String localPath) {
+        return downloadRemoteFile(metadata.getRev(), localPath);
+    }
+    
+    private static File downloadRemoteFile(String revision, String localPath) {
+        DbxClientV2 client = getDbxClient();
         
         FileOutputStream outputStream = null;
         File outputFile = new File(localPath);
         
         try {
             outputStream = new FileOutputStream(outputFile);
-            client.getFile(remoteEntry.getPath(), remoteEntry.getRevision(), outputStream);
+            client.files().download(localPath, revision).download(outputStream);
```

Comment:
The argument to download method probably shouldn't be local path, but the path from DbxEntryDto